scripts/run-integration-tests.sh

```diff
@@ -173,98 +132,80 @@ export KUBECONFIG=$KUBECONFIG_PATH
 # Use should_execute in common.sh to short circuit methods if $TEST_PASS -eq 1
 
 add_helm_repo
-install_helm_chart base
-ensure_controller_pods
+BASE_INTEGRATION_DURATION=0
 if [[ "$TEST_PASS" -ne 0 ]]; then
   echo "NOTE: Skipping base test run because test is marked as failed"
+elif [[ -z "${BASE_GIT_TAG+x}" ]]; then
+  echo "NOTE: Skipping base test run because BASE_GIT_TAG is not given"
 else
-  echo "Running base integration test"
-  # TODO: RUN BASE TEST HERE
```

Comment:
You should move the `for` loop over the contents of the `services/` directory here and for each of the `echo` calls below, tack on the name of the service that you are running integration tests for. Make sense?